src/lib.rs

```diff
@@ -60,9 +60,15 @@ impl Monitor {
 
     /// Physical monitor description string.
     pub fn description(&self) -> String {
-        unsafe {
-            WideCString::from_ptr_str(self.monitor.szPhysicalMonitorDescription.as_ptr())
-                .to_string_lossy()
+        let str_ptr = ptr::addr_of!(self.monitor.szPhysicalMonitorDescription);
+        // TODO: Replace with is_aligned() once it's stable
+        let is_aligned = (str_ptr as usize) & (mem::align_of::<u16>() - 1) == 0;
+        if is_aligned {
+            unsafe { WideCStr::from_ptr_str(str_ptr as _) }.to_string_lossy()
+        } else {
+            let aligned = unsafe { ptr::read_unaligned(str_ptr) };
+            let wide_str = unsafe { WideCStr::from_ptr_str(aligned.as_ptr()) };
+            wide_str.to_string_lossy()
```

Comment:
I'm pretty sure this can just be done safely instead:

```suggestion
            let aligned = self.monitor.szPhysicalMonitorDescription;
            let wide_str = WideCStr::from_slice_truncate(&aligned).expect("do we know if winapi strings are guaranteed to be nul-terminated (thus max length 127) or not (need to use WideStr instead)?");
            wide_str.to_string_lossy()
```